builder/dockerfile/builder.go

```diff
@@ -383,39 +416,33 @@ func BuildFromConfig(config *container.Config, changes []string) (*container.Con
 	b.Stderr = ioutil.Discard
 	b.disableCommit = true
 
-	if err := checkDispatchDockerfile(dockerfile.AST); err != nil {
-		return nil, err
+	commands := []interface{}{}
```

Comment:
now that every command embeds withNameAndCode, I suppose I can introduce an interface yes